systemservices/ethwallet/ethwallet/export.go

```diff
@@ -33,5 +25,24 @@ func (s *Ethwallet) export(input map[string]interface{}) (map[string]interface{}
 	if err = json.Unmarshal(keyJSON, &accountJSON); err != nil {
 		return nil, err
 	}
-	return client.Marshal(accountJSON)
+
+	return types.NewStruct(map[string]*types.Value{
```

Comment:
Let's export and import the account as JSON string instead of object! this will simplify the service and give more time to fine a proper solution.